src/com/apeshko/javacore/task_06/Application.java

```diff
@@ -0,0 +1,67 @@
+package com.apeshko.javacore.task_06;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class Application {
+    public static Annotation getAnnotation(Annotation[] annotations, String annotationName) {
+        for (Annotation annotation : annotations) {
+            if (annotation.annotationType().getName().contains(annotationName)) {
+                return annotation;
+            }
+        }
+
+        return null;
+    }
+
+    public static String getReviewer(Annotation annotation) {
+        try {
+            if (annotation.annotationType().getName().contains("ThisCodeSmellsContainer")) {
+                String reviewers = "";
+                for (Annotation innerAnnotation : (Annotation[]) annotation.getClass().getMethod("value").invoke(annotation)) {
+                    reviewers += getReviewer(innerAnnotation) + " ";
+                }
+
+                return reviewers;
+            }
+
+            if (annotation.annotationType().getName().contains("ThisCodeSmells")) {
+                return (String) annotation.getClass().getMethod("reviewer").invoke(annotation);
+            }
+        } catch (IllegalAccessException e) {
+            e.printStackTrace();
+        } catch (InvocationTargetException e) {
+            e.printStackTrace();
+        } catch (NoSuchMethodException e) {
+            e.printStackTrace();
+        }
+
+        return "";
+    }
+
+    public static void main(String... args) {
+        ClassLoader classLoader = Application.class.getClassLoader();
+
+        Iterable<Class> classes = new ClassesLoader().getClasses(classLoader, "com.apeshko.javacore.task_05");
+
+        for (Class clazz : classes) {
+            if (getAnnotation(clazz.getAnnotations(), "ThisCodeSmells") != null) {
```

Comment:
move this logic to separate service class, like

interface ThisCodeSmellsAnnotationHandler {
  handle(AnnotatedElement element);
}